kong/api/routes/clustering.lua

```diff
@@ -20,4 +23,36 @@ return {
       end,
     },
   },
+
+  ["/clustering/status"] = {
+    schema = kong.db.clustering_data_planes.schema,
+    methods = {
+      GET = function(self, db, helpers)
+        if kong.configuration.role ~= "control_plane" then
+          return kong.response.exit(400, {
+            message = "this endpoint is only available when Kong is " ..
+                      "configured to run as Control Plane for the cluster"
+          })
+        end
+
+        local data = {}
+
+        for row, err in kong.db.clustering_data_planes:each() do
+          if err then
+            kong.log.err(err)
+            return kong.response.exit(500, { message = "An unexpected error happened" })
+          end
+
+          local id = row.id
+          row.id = nil
+
+          data[id] = row
+        end
+
+        return kong.response.exit(200, data, {
```

Comment:
Maybe we can also do

`data["message"] = "This endpoint has been deprecated and will be removed in a future release, please use \"/clustering/data_planes\" instead."` because it seems that the header can be easily missed even when using a CLI based client (such as cURL).

Reply:
I know, but this would break the API contract.